src/schema/__tests__/submission.schema.unit.spec.js

```diff
@@ -146,4 +147,21 @@ describe('Validation Schemas', () => {
       expect(error.details[0].message).toContain('"contact_id" is required')
     })
   })
+
+  describe('getSubmissionBySubmissionIdSchema', () => {
+    it('should validate successfully when "submissionId" is provided and valid', () => {
+      const params = { submissionId: 123 }
+      const { error } = getSubmissionBySubmissionIdSchema.validate(params)
+
+      expect(error).toBeUndefined()
+    })
+
+    it('should return an error if "submissionId" is missing', () => {
+      const params = { submissionId: undefined }
+      const { error } = getSubmissionBySubmissionIdSchema.validate(params)
+
+      expect(error).toBeDefined()
+      expect(error.details[0].message).toContain('"submissionId" is required')
+    })
```

Comment:
strictly speaking, you should also test that a non-numeric value can't be submitted, either